csrankings-p.csv

```diff
@@ -238,6 +238,7 @@ Patrick D. McDaniel,University of Wisconsin - Madison,http://www.patrickmcdaniel
 Patrick D. Pedrow,Washington State University,http://www.eecs.wsu.edu/~pedrow,8FD3lhEAAAAJ
 Patrick De Causmaecker,KU Leuven,https://www.kuleuven-kulak.be/~pdecausm,IHAmK_gAAAAJ
 Patrick Delfmann,University of Koblenz-Landau,https://www.uni-koblenz-landau.de/de/koblenz/fb4/iwvi/p_delfmann/team/patrick-delfmann/patrick-delfmann,ROhACNYAAAAJ
+Patrick Donnelly,Oregon State University,https://osucascades.edu/people/patrick-donnelly,TFDLhm4AAAAJ

```

Comment:
Only full-time faculty at the main campus are eligible for inclusion.

Reply:
I removed Patrick and squashed the remaining commits to keep the commit log neater.